op/upgrade_v3.go

```diff
@@ -0,0 +1,70 @@
+package op
+
+import (
+	"context"
+	"strings"
+
+	"github.com/cybozu-go/cke"
+)
+
+type v3UpgradeOp struct {
```

Comment:
I don't think `v3UpgradeOp` is needed.

The original `upgradeOp` can determine what to do for the current cluster version.